orca-queue/orca-queue.gradle

```
@@ -38,5 +38,7 @@ dependencies {
   testImplementation(project(":orca-test-kotlin"))
   testImplementation(project(":orca-queue-tck"))
   testImplementation(project(":orca-queue-redis"))
+  testImplementation(project(":orca-echo"))
   testImplementation("org.assertj:assertj-core")
+  testImplementation("org.codehaus.groovy:groovy-all:3.0.12")
```

I tried taking this out locally and the test still ran fine.  If there's something I missed and we really do need this, please use `org.codehaus.groovy:groovy-all:2.5.14` to match what the rest of orca (and kork) uses.

---

OK, after a few more iterations, it appears something like this really is required.  My new, improved suggestion is:
```suggestion
  testImplementation("org.codehaus.groovy:groovy")
```

---

Removed